series.tex

```diff
@@ -51,6 +51,7 @@ \subsubsection{Series Expansion}
 \texttt{sympy.polys.rs\_series} takes as input any SymPy expression and hence
 there is no need to explicitly create a polynomial \texttt{ring}. An example:
 
+%no-doctest
```

Comment:
Doctests are skipped for `rs_series` in sympy documentation, so I assumed this is a known issue